docs/source/techniques/CrystalFieldExamples.rst

```diff
@@ -0,0 +1,222 @@
+.. _Crystal field:
```

Comment:
Thanks for fixing the links. Can we also rename this file and the heading to just `Crystal Field` so it's consistent with the other concept pages?